src/hearings/services/hearings-data.service.ts

```diff
@@ -0,0 +1,13 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { RefDataModel } from 'api/hearings/models/refData.model';
```

Comment:
copy api/hearings/models/refData.model into src/hearings/models/refData.model, node models and ng models need to be separated.